package.json

```diff
@@ -56,25 +56,25 @@
     }
   ],
   "engines": {
-    "node": ">= 4"
+    "node": ">= 20"
   },
   "devDependencies": {
-    "consolidate": "^0.15.1",
-    "dustjs-linkedin": "^2.6.2",
-    "ejs": "^2.3.1",
+    "consolidate": "^0.16.0",
+    "dustjs-linkedin": "^3.0.1",
+    "ejs": "^3.1.10",
     "express": "^4.8.4",
     "istanbul": "^0.4.5",
-    "jade": "^1.9.2",
     "jshint": "^2.6.3",
-    "supertest": "^4.0.2",
-    "tape": "^4.0.0"
+    "pug": "^3.0.3",
+    "supertest": "^7.1.4",
+    "tape": "^5.9.0"
   },
   "peerDependencies": {
     "express": "^4.8.4"
   },
   "dependencies": {
     "bluebird": "^3.4.7",
-    "body-parser": "^1.12.2",
+    "body-parser": "^2.2.0",
```

Comment:
Given the current express constraints (express >= 4.8.4), is it possible to use `express.static` and `express.json()`/`express.urlencoded()` instead?